src/Paysera/WalletApi/Client/WalletClient.php

```diff
@@ -1017,4 +1017,13 @@ public function deleteTransferConfiguration($clientId)
 
         return $this->mapper->decodeClient($responseData);
     }
+
+    /**
+     * @param $clientId
+     * @return mixed|null
+     */
+    public function getWalletsPermissions($clientId)
+    {
+        return $this->get(sprintf('client/%s/wallets-permissions', $clientId));
```

Comment:
why data are not mapped here?

There should be some permissions Entity. Permissions might have also more information, then only the scope itself and I doubt it's also important permissions type.


We have  Paysera_WalletApi_Entity_ClientPermissions maybe we could do something similar, maybe better, but it should be not just mixed|null returned

Reply:
There is mixed data of available wallets and existing permissions and looks like
[{"id": 1, "account": "EVP1","balance": false,"statement": false}, {"id": 2, "account": "EVP2","balance": true,"statement": false}, {"id": 3, "account": "EVP3","balance": true,"statement": true}]

Reply:
I've solved it and added mapper.